### PR TITLE
Standarize use of variable colors and how to show close button

### PR DESCRIPTION
**Pull request by nelidita**
The names of the color constants were modified and the repeated constants were eliminated, the burnt colors were replaced by those of the constants, the .svg image of the modal close button was changed to the fontawesome icon and it was modified to flex , The padding of manners was unified at 20px

### CODE REVIEW

File: src/containers/spec-create-product/SpecCreateProduct.styles.jsx [hunk 1]
```diff
@@ -34,28 +34,28 @@ export const Loading = styled.div`
 
 export const Header = styled.section`
   align-items: center;
-  border-bottom: 1px solid rgba(151, 151, 151, 0.32);
```
Comment (Sophia-Sergio) [inline src/containers/spec-create-product/SpecCreateProduct.styles.jsx:37]: how are you managing the opacity here?

**Comment by nelidita** [comment]
Modify and add names of color constants, replace hardcoded colors with the names of their constants, change the image to the close icon of fontawesome, delete image close.svg and delete the imports made in each file

File: src/containers/spec-images-modal/SpecImagesModal.container.jsx [hunk 2]
```diff
@@ -6,72 +6,82 @@ import { onHideSpecImagesModalSuccess } from './SpecImagesModal.actions';
...
 const SpecImagesModal = () => {
-  const { id: specID } = useParams();
-  const { blocks } = useSelector(state => state.specDocument);
-  const { selectedBlockID, show } = useSelector(state => state.specImagesModal);
-  const dispatch = useDispatch();
-  const selectedProductBlock = blocks.find(block => block.id === selectedBlockID) || {};
-  const [selectedImage, setSelectedImage] = useState(selectedProductBlock.element?.product_block_image || '');
-  const { onClose: handleClose } = useModal({ closeCallback: () => dispatch(onHideSpecImagesModalSuccess()) });
-  const handleSelectImage = selected => () => {
+	const { id: specID } = useParams();
```
Comment (Sophia-Sergio) [inline src/containers/spec-images-modal/SpecImagesModal.container.jsx:28]: este archivo está con una identación que no corresponde.

File: src/components/attachments/AttachedDocuments.jsx [hunk 3]
```diff
@@ -158,7 +157,7 @@ const AttachedFiles = (props) => {
 			)}
 			<ModalLayout show={show} onClose={handleClose}>
 				<DropContent>
-					<DropCloseIcon alt="" src={closeSource} onClick={handleClose} />
+					<CloseIcon alt="" className="fas fa-times" onClick={handleClose} />
```
Comment (Sophia-Sergio) [inline src/components/attachments/AttachedDocuments.jsx:160]: sobre los `CloseIcon`, algunos tienen el props `alt` y otros no. Standarizar, sugiero que no sea una props y que alt sea `Cerrar` por defecto. Qué te parece?

File: src/components/attachments/AttachedDocuments.styles.jsx [hunk 4]
```diff
@@ -1,209 +1,213 @@
...
+	GALLERY,
+	MINE_SHAFT,
+	MINE_SHAFT_RGB,
+	SILVER_CHALICE,
+} from '../../config/constants/styled-vars';
 
 export const Root = styled.div`
-  position: relative;
-  width: 100%;
+	position: relative;
```
Comment (Sophia-Sergio) [inline src/components/attachments/AttachedDocuments.styles.jsx:17]: recuerda solo incoporar cambios reelevantes, no de identación a menos que sea necesario.
por ejemplo acá la identación era correcta antes, y ahora la cambiaste a 4 espacios.

File: src/components/layouts/ModalLayout.styles.jsx [hunk 5]
```diff
@@ -1,24 +1,39 @@
...
+`;
+
+export const CloseIcon = styled.i`
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+	font-size: 24px;
+	cursor: pointer;
+	outline: none;
+	// width: 100%;
```
Comment (Sophia-Sergio) [inline src/components/layouts/ModalLayout.styles.jsx:28]: remove commented line

File: src/containers/spec-modal-quote/SpecModalQuote.container.jsx [hunk 6]
```diff
@@ -33,154 +38,154 @@ import { useFormik } from 'formik';
...
 const SpecModalQuote = ({initialValues}) => {
-  const dispatch = useDispatch();
-  let isMounted = true;
-  const getFirstImg = data => (data?.images?.length && data.images[0]) || {};
-  const { product, showModalQuote } = useSelector(state => state.specModalQuote);
-  const { project } = useSelector((state) => state.specDocument);
-  const [selectedImg, selectImg] = useState(getFirstImg());
-  const onSelectImg = img => () => selectImg(img);
+	const dispatch = useDispatch();
+	let isMounted = true;
```
Comment (Sophia-Sergio) [inline src/containers/spec-modal-quote/SpecModalQuote.container.jsx:45]: acá declaraste esta variable y no la ocupas en ni una lado

File: src/containers/spec-modal-quote/SpecModalQuote.container.jsx [hunk 7]
```diff
@@ -33,154 +38,154 @@ import { useFormik } from 'formik';
...
+	const getFirstImg = data => (data?.images?.length && data.images[0]) || {};
+	const { product, showModalQuote } = useSelector(state => state.specModalQuote);
+	const { project } = useSelector((state) => state.specDocument);
+	const [selectedImg, selectImg] = useState(getFirstImg());
+	const onSelectImg = img => () => selectImg(img);
```
Comment (Sophia-Sergio) [inline src/containers/spec-modal-quote/SpecModalQuote.container.jsx:50]: acá también declaras una variable pero no la usas

**Review by Sophia-Sergio — APPROVED** [review]
LGTM

**Pull request merged**